components/telemetry-operator/README.md

```diff
@@ -67,3 +67,60 @@ make deploy-local IMG_NAME=<my container repo> TAG=latest
 ```bash
 make undeploy-local
 ```
+## Trying it out on a Kyma cluster
+
+### Prerequisites
+
+- A Kyma Cluster with the latest installation of kyma
+
+### Enable Telemetry Operator integration
+```bash
+kyma deploy --component=telemetry 
+```
+
+The above command would perform following tasks
+- installs telemetry operator
+- Installs Telemetry fluent-bit daemon set with null output config
+
+After installing the telemetry operator, it is necessary to configure the telemetry fluent-bit component to push logs to the Loki backend (which is already present on the Kyma cluster)
+
+```bash
+kyma deploy --component logging --value global.telemetry.enabled=true
+```
+
+The above command would additionally install a LogPipeline CR which would configure the telemetry-fluent-bit to push logs to the Loki. After disabling the fluent-bit from logging chart we need delete the unneeded resources
+
+```bash
+kubectl delete daemonset -n kyma-system logging-fluent-bit
+kubectl delete configmap -n kyma-system logging-fluent-bit
+kubectl delete servicemonitor -n kyma-system logging-fluent-bit
+```
+
+
+### Disabling Telemetry Operator integration
+```bash
+kyma deploy --component logging 
+```
+
+The above command would redeploy the fluent-bit from the logging chart. After installing the chart we can disable the LogPipeline CR so that it does not collect the logs anymore
+
+```bash
+kubectl delete validatingwebhookconfigurations validation.webhook.telemetry.kyma-project.io
```

Comment:
This uninstalls the complete chart, right? Why don't we just do `kyma undeploy --component telemetry`? For disabling the telemetry operator integration, wouln't we just have to delete the CR `k delete k delete logpipelines.telemetry.kyma-project.io loki`?

Reply:
`kyma undeploy` with a component list seem to have a problem with PVCs at the moment. I suggest to keep this for now and open an issue for the `undeploy` first.